collector/mongod/replset_status.go

```diff
@@ -1,232 +1,223 @@
 package collector_mongod
 
 import (
-    "time"
-    "github.com/golang/glog"
-    "github.com/prometheus/client_golang/prometheus"
-    "gopkg.in/mgo.v2"
-    "gopkg.in/mgo.v2/bson"
+	"time"
+
+	"gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+
+	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus"
 )
 
 var (
-    replSetLastElection = prometheus.NewCounterVec(prometheus.CounterOpts{
-            Namespace: Namespace,
-            Name:      "replset_last",
-            Help:      "Last event times for replica set",
-    }, []string{"event"})
-    replSetTotalMembers = prometheus.NewGauge(prometheus.GaugeOpts{
-            Namespace: Namespace,
-            Subsystem: "replset",
-            Name:      "members",
-            Help:      "Number of members in replica set",
-    })
-    replSetTotalMembersWithData = prometheus.NewGauge(prometheus.GaugeOpts{
-            Namespace: Namespace,
-            Subsystem: "replset",
-            Name:      "members_w_data",
-            Help:      "Number of members in replica set with data",
-    })
-    replSetMyLagMs = prometheus.NewGauge(prometheus.GaugeOpts{
-            Namespace: Namespace,
-            Subsystem: "replset",
-            Name:      "my_lag_ms",
-            Help:      "Lag in milliseconds in reference to replica set Primary node",
-    })
-    replSetMaxNode2NodePingMs = prometheus.NewGauge(prometheus.GaugeOpts{
-            Namespace: Namespace,
-            Subsystem: "replset",
-            Name:      "max_n2n_ping_ms",
-            Help:      "Maximum ping in milliseconds to other replica set members",
-    })
+	myState   = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: Namespace,
+		Subsystem: "replset",
+		Name:      "my_state",
+		Help:      "An integer between 0 and 10 that represents the replica state of the current member",
+	}, []string{"set"})
+	term = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
The area I see highlighted here is "myState" which seems to be a good name to me. This is upstream code so I tried not to mess with it, but I am welcome to change this to anything if it doesn't cause incompatibility with the graph templates